tez-dag/src/main/java/org/apache/tez/dag/api/client/DAGStatusBuilder.java

```diff
@@ -61,6 +61,13 @@ public void addVertexProgress(String name, ProgressBuilder progress) {
     getBuilder().addVertexProgress(builder.build());
   }
 
+  //TODO: let this be a map of values in protobuf 3.x
```

Comment:
What is this todo for? or am i missing something here?

Reply:
I wanted to put these memory related stuff into a map, which can be dinamically extended later with other metrics if needed, but unfortunately the protoc compiler which is used in protobuf 2.5 doesn't support maps